File: Java/bee 1012.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a, b, c;
        a = sc.nextDouble();
        b = sc.nextDouble();
        c = sc.nextDouble();

        System.out.printf("TRIANGULO: %.3f\n", (a * c)/2);
        System.out.printf("CIRCULO: %.3f\n", 3.14159 * Math.pow(c,2));
        System.out.printf("TRAPEZIO: %.3f\n", ((a + b) * c)/2);
        System.out.printf("QUADRADO: %.3f\n", Math.pow(b,2));
        System.out.printf("RETANGULO: %.3f\n", a*b);
    }
}

File: Kotlin/bee 1012.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.pow

fun main(args: Array<String>){
    val br = BufferedReader(InputStreamReader(System.`in`))
    val valores = br.readLine().split(" ")
    val a = valores[0].toDouble()
    val b = valores[1].toDouble()
    val c = valores[2].toDouble()

    println(String.format("TRIANGULO: %.3f", (a*c)/2))
    println(String.format("CIRCULO: %.3f", 3.14159 * c.pow(2)))
    println(String.format("TRAPEZIO: %.3f", ((a+b)*c)/2))
    println(String.format("QUADRADO: %.3f", b.pow(2)))
    println(String.format("RETANGULO: %.3f", a*b))
}
